rust/basic_bitcoin/README.md

```diff
@@ -214,6 +214,26 @@ it sent to the network. You can track the status of this transaction using a
 transaction has at least one confirmation, you should be able to see it
 reflected in your current balance.
 
+## Step 6: Retrieving block headers
+
+You can also get range of Bitcoin block headers by using the `get_block_headers` 
+endpoint on your canister.
+
+In the Candid UI, write the desired start height and optionally end height, and click on "Call":
+
+Alternatively, make the call using the command line. Be sure to replace `10` with your 
+desired start height:
+
+```bash
+dfx canister --network=ic call basic_bitcoin get_block_headers "(10: nat32)"
+```
+or replace `0` and `11` with your desired start and end height respectively:
+```bash
+dfx canister --network=ic call basic_bitcoin get_block_headers "(0: nat32, 11: nat32)"
+```
+
+Checking the balance of a Bitcoin address relies on the [bitcoin_get_block_headers](https://internetcomputer.org/docs/current/references/ic-interface-spec/#ic-bitcoin_block_headers) API.
```

Comment:
Is this really true? Seems like a copy-paste error to me.

Reply:
It is not, I will remove it.